### ISSUE TITLE
mess with hashides

### ISSUE THREAD

**Issue by gmhurley** [gmhurley/urly-bird#12]

**Comment by gmhurley** [comment]
needed to get short urls.

**Comment by gmhurley** [comment]
So I've messed around with this for a little over an hour tonight, and I think I've wrapped my head around what we need to do. 

- All of our URLs are going to be stored in the short table. 
- When a URL is entered, we need to lookup the last record on the short table ```Short.objects.last()```
- That object.id + 1 will be the int that we use to encode a new URL
- we can then use hashid:
```
>>> hashids = Hashids(salt='tiyd python 2015-08')
>>> hashid = hashids.encode(1)
>>> hashid
'kB'
```
- We'll store the value returned ('kB') in this example  in the short.short_url column.
- The original url, ```str(request.path)``` will be stored in the short.bookmark column.

When urls come into the site (localhost:8000/kB) we'll be able to filter the short table to find kB and send the user to the value put in bookmark. At the same time, we should be able to capture all the info from the redirect.
